modules/aws_base/aws-base.md

```diff
@@ -9,3 +9,14 @@ description: Sets up VPCs, a default KMS key, and the db/cache subnets for your
 
 The defaults for this module are set to work 99% of the time, assuming no funny networking constraints (you'll know them
 if you have them), so in most cases, there is _no need to set any of the fields or know what the outputs do_.
+
+## Bring your own VPC
+To use an existing VPC with Opta, instead of having Opta create a new VPC, you must set the `vpc_id`, `public_subnet_ids`, and `private_subnet_ids` fields.
+Set `vpc_id` to the ID of the existing VPC you would like Opta to use.
+Set `public_subnet_ids` to the list of IDs of the public subnets in the VPC.
+Public subnets must have a route that connects to an internet gateway, and must be configured to assign public IP addresses.
+Set `private_subnet_ids` to the list of IDs of the private subnets in the VPC.
+Private subnets must have a route with a destination of `0.0.0.0/0` that points to a NAT gateway with a public IP address.
```

Comment:
suggestion: For the must-have listed here, what would happen if they don't have them?
would terraform fail in a way that inform the user about what's wrong?
we could also consider a pre hook to check for the requirements so we can provider a user with a clear error message if the terraform output will not be enough. 

if it's too complicated to do such thing, I think we can skip it, this documentation is pretty clear.

Reply:
Ok, may be we can just _document_ what to do if the user gets this error message, to save us opta support on it.